sts-mock/src/token/tokenEncrypter/tests/tokenEncrypter.test.ts

```diff
@@ -0,0 +1,220 @@
+import { TokenEncrypter } from "../tokenEncrypter";
+
+describe("Token Encrypter", () => {
+  let tokenEncrypter: TokenEncrypter;
+
+  const dummyJwt = "header.payload.signature";
+
+  beforeEach(() => {
+    tokenEncrypter = new TokenEncrypter("dummyJwksUrl");
```

Comment:
Small nit but would you prefer the dummyJwksUrl to be in the shape of a URL as that's what it is meant to be? e.g. https://dummyJwksUrl.com? Appreciate it doesn't affect the behavior of the test though so happy for you to resolve if you don't see enough value

Reply:
Although might affect the behavior if you do change your config service to check if it's a valid URL

Reply:
We've been bitten (gently) once before with this when we thought we had regression coverage using our unit tests but our test data wasn't in the right format. Given it's hopefully very low effort - please may we structure this as a URL? I appreciate this doesn't change the behaviour of our tests/codebase at all though!